tests/server/test_server.py

```diff
@@ -113,6 +113,33 @@ async def test_solve_missing_predicate():
     output = await sync_query(q)
 
 
+@pytest.mark.asyncio
+@with_translator_overlay(
+    settings.kpregistry_url,
+    settings.normalizer_url,
+    [
+        ("ctd", CTD_PREFIXES),
+        ("hetio", DEFAULT_PREFIXES),
+        ("mychem", MYCHEM_PREFIXES),
+    ])
+async def test_solve_missing_category():
+    """Test solving the ex1 query graph, in which one of the categories is missing. """
```

Comment:
For a test that covers a very specific condition like this, I think it's more readable to keep a simple query graph inside the test itself:
```
n0(( id MONDO:0005148 ))
n1(( category biolink:Drug ))
n0-- biolink:treated_by -->n1
```

Reply:
I agree. I was following the format that @cbizon used in the "missing predicate" test. At some point, we can go back and make both of these tests more readable.